event.py

```diff
@@ -1,20 +1,25 @@
+"""
+Implementing CalicoBot events
+"""
 import discord
 import datetime
 from discord.ext import commands
-from discord import app_commands
 import time
-import asyncio
-import difflib
-import main
 
-bot = main.bot
+bot = None
+def event_init(main_bot):
```

Comment:
Very sorry,I forgot this description.
```suggestion
def event_init(main_bot):
     """
     A function to initialize event module

    Args:
        main_bot (discord.bot): set your bot instance to get channels
     """
```